### PR TITLE
Adjust bugs on analysis files

### PR DESCRIPTION
**Pull request by ecastrow**
In the master branch of my fork, yes.

### CODE REVIEW

File: analysis/simtb_analysis.py [hunk 1]
```diff
@@ -144,7 +144,7 @@ def main(model, out_path=None, prefix=None, **anal_args):
         dataset = dataset.raw
 
     ms = fe.ModelStructure(model, dataset)
-    data_path = serial.preprocess(dataset.dataset_root + dataset.dataset_name)
+    data_path = serial.preprocess(dataset.dataset_root+'/'+dataset.dataset_name)
```
Comment (rdevon) [inline analysis/simtb_analysis.py]: FYI, keep spaces between binary ops for readability. Also use double quotes. Make those changes for me please and push again.

serial.preprocess(dataset.dataset_root + "/" + dataset.dataset_name)
I typically follow this:
https://google-styleguide.googlecode.com/svn/trunk/pyguide.html

**Comment by rdevon** [comment]
Did these get fixed in the master?

**Comment by karmeze** [comment]
Hi!
So I am testing your scripts to use in my labs research with Deep Learning, and I am having trouble running the tutorials.  Some of the scripts seem to be missing, i.e. /tools/mri_analysis and /dataset_utils/mri_nifiti
Have they been relocated to a different file?

Thanks for the help!

**Comment by rdevon** [comment]
Hi there. pl2mind is currently out of development, but a newer, easier to use toolbox is in the works and is being tested in-house. We are hoping that a beta invite-only version will be rolling out within the month, and we would be happy to give you access.

Just an aside, pl2mind is based on pylearn2. pylearn2 is incredibly bulky and difficult to debug, and is not even used at LISA, where it resides. The primary issue is that pylearn2 is class and inheritance heavy, but also tries to hide all the underlying Theano. The end-results is if anything goes wrong, it's impossible to debug because it can be intertwined with various inheritances. The new toolbox has very shallow class structure, is easy to customize, and relatively easy to debug.

As far as those scripts, I don’t recall what happened to mri_nifti, but mri_analysis can be found here:
https://github.com/MRN-Code/pl2mind/tree/master/analysis

-devon

**Pull request closed without merging**